app/uk/gov/hmrc/minorentityidentificationfrontend/controllers/trustControllers/CaptureCHRNController.scala

```diff
@@ -21,7 +21,6 @@ import uk.gov.hmrc.auth.core.retrieve.v2.Retrievals.internalId
 import uk.gov.hmrc.auth.core.{AuthConnector, AuthorisedFunctions}
 import uk.gov.hmrc.http.InternalServerException
 import uk.gov.hmrc.minorentityidentificationfrontend.config.AppConfig
-import uk.gov.hmrc.minorentityidentificationfrontend.controllers.overseasControllers.{routes => overseasControllersRoutes}
 import uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.{routes => trustControllersRoutes}
```

Comment:
I expect the same applies here. I think you only need to create the "overseas" and "trusts" routes if you need to redirect to controllers in different packages in the same class.

Reply:
cool okay I will do it.